src/ktn_os.erl

```diff
@@ -0,0 +1,49 @@
+%% @doc Utility functions to run commands in the underlying OS.
+-module(ktn_os).
+
+-export([command/1, command/2]).
+
+-type opts() :: #{log_fun => function()}.
```

Comment:
can you improve the type of `log_fun`, using something like `fun((iodata()) -> _)` (or whatever applies) instead of `function()` ?